### ISSUE TITLE
Sort operation makes separate network request for each rank update

### ISSUE THREAD

**Issue by jpizzle34** [plantagoIT/strapi-drag-drop-content-type-plugin#17]
A sort action, which involves moving  an entry `n` places, will result in `n` http requests being made in serial. For example: moving an item from rank 1 to rank 100 will send 100 http requests one by one. Network requests are costly; how can we batch rank updates into a single network request?

https://github.com/plantagoIT/strapi-drag-drop-content-type-plugin/blob/7b2da27c7f39163106c25d003dc0db61648044fe/admin/src/components/SortModal/index.js#L120-L137

**Comment by derschiw** [comment]
Yes this in fact is an issue. It would be way nicer to send the array as `json` via `POST` and then loop on the server.
